src/Common/src/TypeSystem/IL/Stubs/PInvokeMarshallingILEmitter.cs

```diff
@@ -65,6 +65,11 @@ public static bool RequiresMarshalling(MethodDesc method)
                 }
             }
 
+            if (UseLazyResolution(method, method.GetPInvokeMethodMetadata().Module))
+            {
+                return true;
+            }
+
```

Comment:
(Sigh, maybe `PInvokeMarshallingILEmitter` should be just `PInvokeILEmitter` too.)